src/fragments/lib/in-app-messaging/js/display-message.mdx

```diff
@@ -0,0 +1,33 @@
+In-app messages are displayed when an In-App Messaging or analytics event is sent and matches the criteria set forth by your active In-App messaging campaigns.
+
+## Analytics event
+
+Now that messages have been synced to your app’s users’ devices, Amplify In-App Messaging will allow you to start displaying them with Amplify Analytics events with no additional integration steps. Any events you record or are already recording using the `Analytics.record` API are automatically picked up and processed by In-App Messaging. If the event matches the attributes and criteria defined in an in-app message, that message will be displayed.
+
+```js
+const event = {
+  name: 'first_event',
+  attributes: { color: 'red' },
+  metrics: { quantity: 10 }
+};
+
+Analytics.record(event);
+```
+
+If the event name, attributes, and metrics match the campaign you have setup in Pinpoint, you will be able to see the in-app message in your app.
+
+## In-App Messaging event
+
+In addition to or instead of Amplify Analytics events, you can also dispatch In-App Messaging events to trigger an in-app message display programmatically.
+
+```js
+const event = {
+  name: 'first_event',
+  attributes: { color: 'red' },
+  metrics: { quantity: 10 }
+};
+
+InAppMessaging.dispatchEvent(event);
+```
+
+If the event name, attributes, and metrics match the campaign you have setup in Pinpoint, you will be able to see the in-app message in your app.
```

Comment:
Nit: Want to avoid talking about Pinpoint directly if possible as they are one of our plugins, not necessarily the only one forever.
```suggestion
If the event name, attributes, and metrics match those set forth by one of your In-App Messaging campaigns, you should see the in-app message displayed in your app.
```